src/clj/rems/application/model.clj

```diff
@@ -664,11 +664,15 @@
         visible? (comp visible-ids :attachment/id)]
     (update application :application/attachments #(filterv visible? %))))
 
+(defn user-is-applicant-or-member [application user-id]
+  (let [roles (permissions/user-roles application user-id)]
+    (or (contains? roles :applicant)
+        (contains? roles :member))))
+
 (defn see-application? [application user-id]
-  (let [permissions (permissions/user-roles application user-id)]
-    (if (and (= :application.state/draft (:application/state application)) (contains? permissions :reporter))
-      false
-      (not= #{:everyone-else} permissions))))
+  (if (= :application.state/draft (:application/state application))
+    (user-is-applicant-or-member application user-id)
+    (not= #{:everyone-else})))
```

Comment:
missing the other argument to `not=` here :)

Reply:
;D